run-profile

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -x
+set -e
+
+# Assume that assignments have been generated into ../zeth/build
+
+ASSIGNMENTS_DIR=../zeth/build
+ZETH_TOOL=../zeth/build/zeth_tool/zeth-tool
+
+# 1 - jsin/jsout value
+function setup_files() {
+    # Unpack data from the archive, either from
+    # ${STORE_HOST}:${STORE_PATH}, or the local ${ASSIGNMENTS_DIR}
+    if ! [ -e assignment_$1_$1.bin ] ; then
+        if [ "${STORE_HOST}" != "" ] ; then
+	        ssh ${STORE_HOST} cat ${STORE_PATH}/zeth_assignment_$1_$1.tar.bz2 |
+                tar -vxjf -
+        else
+            tar -vxjf ${ASSIGNMENTS_DIR}/zeth_assignment_$1_$1.tar.bz2
+        fi
+    fi
+    if ! [ -e assignment_$1_$1.bin ] ; then
+        echo no file assignment_$1_$1.bin
+        exit 1
+    fi
+
+    # If necessary, extract vk from the keypair
+    if ! [ -e pk_$1_$1.bin ] ; then
+        ${ZETH_TOOL} split-keypair --pk-file pk_$1_$1.bin keypair_$1_$1.bin
+    fi
+    if ! [ -e pk_$1_$1.bin ] ; then
+        echo no file pk_$1_$1.bin
+        exit 1
+    fi
+}
+
+# 1 - jsin/jsout value
+function remove_files() {
+    rm *_$1_$1.bin
+}
+
+# 1 - jsin/jsout value
+# 2 - iteration
+function profile_for_jsinout_iteration() {
+    echo "=============================================="
+    echo "  PROFILE: $1_$1 ITERATION: $2"
+    echo "=============================================="
+    time ./run-prover pk_$1_$1.bin assignment_$1_$1.bin > profile_$1_$1_$2.txt 2>&1
+}
+
+# 1 - jsin/jsout value
+function profile_for_jsinout() {
+    echo "=============================================="
+    echo "  PROFILE: $1_$1"
+    echo "=============================================="
+
+    setup_files $1
+
+    profile_for_jsinout_iteration $1 1
+    profile_for_jsinout_iteration $1 2
+    profile_for_jsinout_iteration $1 3
+
+    remove_files $1
+}
+
+# Run for a single JSIN/OUT value, or run over all values.
+if [ "" != "$1" ] ; then
```

Comment:
`if [ "" != "$1" ]` = `if [ "$1" != "" ]`

Reply:
order swapped in latest commit.